resources/features/Sampling.feature

```diff
@@ -1,73 +1,146 @@
         When a <language> Sample is created from a Stratified Frame
-        Then a Sample DataFrame containing the Prn-Sampling strata is returned and exported to CSV
+        Then a Sample containing the sample selection from the Prn-Sample strata is returned and exported to CSV
+            | ern        | entref     | name                          | tradingstyle | address1                 | address2       | address3    | address4                | address5 | postcode | legalstatus | sic07 | paye_empees | paye_jobs | ent_turnover | std_turnover | grp_turnover | cntd_turnover | app_turnover | prn         | cell_no |
+            | 1100000007 | 9906000275 | IBANOCTRACTS UK LTD           |              | 1 GLYNDE PLACE           | HORSHAM        | WEST SUSSEX |                         |          | RH12 1NZ | 1           | 46120 | 2           | 2         | 287          | 287          | 0            | 0             | 0            | 0.155647458 | 5813    |
+            | 1100000008 | 9906000325 | TLUBARE                       |              | 1 GORSE ROAD             | REYDON         | SOUTHWOLD   |                         |          | IP18 6NQ | 1           | 46130 | 3           | 3         | 197          | 197          | 0            | 0             | 0            | 0.446872271 | 5813    |
+            | 1100000004 | 9906000145 | AUBASOT(CHRISTCHURCH) LIMITED |              | 1 GARTH EDGE             | SHAWFORTH      | WHITWORTH   | ROCHDALE LANCASHIRE     |          | OL12 8EH | 2           | 45320 | 0           | 0         | 7            | 7            | 0            | 0             | 0            | 0.509298879 | 5813    |
+            | 1100000002 | 9906000045 | BUEADLIING SOLUTIONS LTD      |              | 1 HAZELWOOD LANE         | ABBOTS LANGLEY |             |                         |          | WD5 0HA  | 3           | 45190 | 1           | 0         | 100          | 100          | 0            | 0             | 0            | 0.63848639  | 5813    |
 
         @JVM
         Examples:
-        | language | input_path                                 | strat_properties_path                         | output_path                               |
-        | Scala    | ./resources/inputs/sampling/frame_data.csv | ./resources/inputs/sampling/prn_sampling.csv  | ./resources/outputs/sampling/prn_sampling |
-
-
-    Scenario Outline: Stratas contain Sample Size parameter that are out of bounds of the sorted Frame records.
-        Given a Stratified Frame and a stratification properties file paths:
-            | data_input_path   | strat_properties        | expected_output   |
-            | <input_path>      | <strat_properties_path> | <output_path>     |
+        | language |
+        | Scala    |
+
+
+    Scenario Outline: Strata contains a Sample Size parameter that is out of bounds of the sorted Stratified Frame population.
```

Comment:
Test is for PRN 'wrapping' - The Given, When, Then is the same as a normal PRN selection as you should still get a full output. Or do we add an additional Then statement that reads something like:
"the Sample continues to select the Sample Size restarting from the begining of the PRN line"?

Amend Scenario Outline to read:
Strata contains a Sample Size parameter that exceeds the end of the sorted PRN line.
Amend the Then statement to read:
a Sample containing the Sample Size from the PRN-sampling strata is returned and exported to CSV